c10/util/C++17.h

```diff
@@ -73,48 +73,6 @@ make_unique_base(Args&&... args) {
   return std::unique_ptr<Base>(new Child(std::forward<Args>(args)...));
 }
 
-#if defined(__cpp_lib_logical_traits) && !(defined(_MSC_VER) && _MSC_VER < 1920)
-
-template <class... B>
```

Comment:
for backwards compatability shouldn't this if_def becomes unconditional alias here (for internal meta code and Python extensions that might uses aliases features)?

Reply:
It is actually safer to keep the templates.